openid-federation-1_0.xml

```diff
@@ -9825,7 +9777,13 @@ Host: op.umu.se
 	-40
 	<list style="symbols">
 	  <t>
-	    TBD
+	    Fixed #34:
+	    Replaced <spanx style="verb">request_authentication_methods_supported</spanx>
+	    with <spanx style="verb">pushed_authentication_methods_supported</spanx>.
+	  </t>
+	  <t>
+	    Fixed #98: Required audience when using <spanx style="verb">private_key_jwt</spanx>
+	    to be the Authorization Server's Entity Identifier.
```

Comment:
Currently, this requirement only applies to pushed authorization requests, not to token requests, and not to requests to Federation endpoints with client authentication.

Reply:
I believe I'd updated the audience language and examples everywhere.